docker-compose.yml

```diff
@@ -258,6 +258,19 @@ services:
   # PEcAn models, list each model you want to run below
   # ----------------------------------------------------------------------
 
+  # PEcAn basgra model runner
```

Comment:
it is fine to have it here since the image will automagically be build by the docker.sh file that is part of the pull request.